app/components/notification-list-row/component.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+const frequencyLabel = {
+    instant: 'components.notificationListRow.instant',
+    daily: 'components.notificationListRow.daily',
+    never: 'components.notificationListRow.never',
+};
+
+export default Component.extend({
+    i18n: service(),
+
+    currentSettingLabel: computed('subscription.frequency', function () {
+        const frequency = this.get('subscription.frequency');
+        if (frequency === 'instant') {
```

Comment:
This can also be DRY'd up if you use an array and [findBy](https://emberjs.com/api/ember/2.18/classes/EmberArray/methods/findBy?anchor=findBy) (h/t Abram for findBy).